### PR TITLE
It doesn't work

### PR DESCRIPTION
**Pull request by tokiiiiiiiin**
am sory

hello fren

### CODE REVIEW

File: src/PChecker.cpp [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 //override the function "checkMyLetter"
 //create the function definition here
-
+	char checkMyLetter(char inputChar){char defaultLetter = inputChar;if(inputChar == 'p' || inputChar == 'P'){return 'p';}else{return defaultLetter;}}
```
Comment (KMamuyac) [inline src/PChecker.cpp]: refer to src/LetterChecker.cpp you only need minor changes in the code

**Review by KMamuyac — COMMENTED** [review]
refer to LetterChecker.cpp and LetterChecker.h and then add minor adjustments.

Refer to OChecker or UChecker if you want

**Review by KMamuyac — CHANGES_REQUESTED** [review]
change c:

**Review by tokiiiiiiiin — COMMENTED** [review]
okay na siya pls accep

**Comment by tokiiiiiiiin** [comment]
pls aksep

**Review by KMamuyac — APPROVED** [review]
Nice job

**Comment by KMamuyac** [comment]
boss, squash and merge boss

**Pull request merged**